ReactiveFeedbackTests/SystemTests.swift

```diff
@@ -50,10 +50,10 @@ class SystemTests: XCTestCase {
 
     func test_reduce_with_two_immediate_feedback_loops() {
         let feedback1 = Feedback<String, String> { state in
-            return SignalProducer(value: "_a")
+            return !state.hasSuffix("_a") ? SignalProducer(value: "_a") : .empty
         }
         let feedback2 = Feedback<String, String> { state in
-            return SignalProducer(value: "_b")
+            return !state.hasSuffix("_b") ? SignalProducer(value: "_b") : .empty
```

Comment:
Why is this changed?

Reply:
It has been broken since ReactiveSwift 3.1, which fixed a bug regarding constant producer not being interrupted as expected. So with ReactiveSwift 3.1, the events from the second feedback would never reach the reducer, hence the addition of the condition.